docs/config/enabling-agent-service.rst

```diff
@@ -2,6 +2,8 @@
 Enabling the Flocker agent service
 ==================================
 
+The Flocker agents are the workhorses of Flocker; you have one on each node in your cluster, and enabling them is an essential step in setting up your cluster:
```

Comment:
There are two on each node right now.  There's the dataset convergence agent and the container convergence agent.

The plan is to get rid of the container convergence agent or at least to make it optional (so if you're using something like kubernetes to manage containers, you don't have to run our container management software as well).  We're not there yet, though, and it's not clear when we will be.  Also it's not clear what the installation story will look like at that point.  It could be drastically different from what we have now.

If you want some examples of things the dataset agent does, they're things like "create the filesystem for a newly created dataset" and "mount and unmount filesystems to support movement of datasets between nodes".